apps/gitness/src/AppMFE.css

```diff
@@ -0,0 +1 @@
+@import '@harnessio/ui/styles.css';
```

Comment:
do we need this file? I see that `AppMFE.tsx` is directly importing `@harnessio/ui/styles.css`

Reply:
```ts
import styles from '!!raw-loader!@harnessio/ui/styles.css'
```

This statement is to import the styles as raw strings in order to pass to shadow dom.

In this case, we can test whether importing CSS Variables alone in `AppMFE.css` is suffice or not, excluding all other styles. So that we can prevent importing the styles twice.